cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/LocationGroupEndpointInput.java

```diff
@@ -156,12 +156,12 @@ public static final class Patch extends EndpointInput {
         private final LocationGroup locationGroup;
         private final String originalGroupId;
         private boolean replaceAssignedLocs = false;
-        private final String office;
+        private final String userOffice;
```

Comment:
There is no user office, it's either location, group, or category office

Reply:
Swapped for groupOffice